src/main/scala/com/cloudant/clouseau/IndexManagerService.scala

```diff
@@ -101,7 +101,9 @@ class IndexManagerService(ctx: ServiceContext[ConfigurationArgs]) extends Servic
     val fieldCache = FieldCache.DEFAULT
     var result = 0L
     for (cacheEntry <- fieldCache.getCacheEntries) {
-      result += RamUsageEstimator.sizeOf(cacheEntry.getValue())
+      val value = cacheEntry.getValue().asInstanceOf[Accountable]
```

Comment:
It is the value stored inside a `CacheEntry` for which the type cast is called.  In [the definition of the `CacheEntry` class](https://github.com/apache/lucene/blob/releases/lucene-solr/4.10.4/lucene/core/src/java/org/apache/lucene/search/FieldCache.java#L701-L766), the value is passed as an `Accountable` at the constructor but the `getValue()` method returns an `Object`.  Nothing in the class definition implies a change in the type, that is why I thought it could be safely casted "back".

Reply:
I tried to blow up the code but I could not.  I activated the `clouseau.field_cache.size` metric and read its value through the JMX interface (with the help of JConsole) but it was always zero.  I ran `make mango-test` and `make elixir-search` from CouchDB, and other metrics, such as `lru.misses`, `lru.evictions`, and `opens` did show values greater than zero.

I tend to believe this happens because `CacheEntry/getValue()` always returns `null`.

Reply:
Ah, even better: the length of the array of the field cache entries itself is zero.

Reply:
my bad on this. I glazed over the 'getValue()' call which returns an Object which is also Accountable. 

Please add a test for whether the result of getValue is Accountable (so we don't blow up in a future change) and I think this section is fine.

Reply:
It was eventually decided to remove this metrics all together, see https://github.com/cloudant-labs/clouseau/pull/76 .